x/bep3/client/rest/query.go

```diff
@@ -0,0 +1,36 @@
+package rest
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/kava-labs/kava/x/bep3/types"
+)
+
+func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router) {
+	r.HandleFunc("/bep3/parameters", queryParamsHandlerFn(cliCtx)).Methods("GET")
```

Comment:
Make note to add rest client methods, can be in a follow-on pr

Reply:
Implemented `/bep3/swap` in https://github.com/Kava-Labs/kava/pull/370/commits/ac92e26999cf32a5fae8296ba92a05b1da0cf357. Follow-on PR is still needed to add the other rest client methods.

Reply:
Issue https://github.com/Kava-Labs/kava/issues/384